src/io/sampling.jl

```diff
@@ -131,6 +203,20 @@ function sample_cf(d::Domain, n::Int, sampler=SobolSample())::DataFrame
     return adjust_cf_samples(d, df)
 end
 
+"""
+    adjust_guided_bounds(guided_spec::DataFrame, lower::Int64)::DataFrame
+    
+Adjust lower bound of guided parameter spec to alter sampling range.
+"""
+function adjust_guided_bounds(guided_spec::DataFrame, lower::Int64)::DataFrame
+    guided_col = guided_spec.fieldname .== :guided
+    g_upper = guided_spec[:, :upper_bound][1]
+    guided_spec[guided_col, :val] .= lower
```

Comment:
If we really really need this as a separate function, you could take the opportunity to optimize - update all values in one go.

```julia
guided_spec[guided_col, [:val, :lower_bound, :bounds, :full_bounds]] .= [lower, lower, (lower, g_upper), (lower, g_upper)]
```

My previous implementation was just trying to be clear about what was happening, but updating values one by one is much slower.

Reply:
it doesn't have to be a separate function, it's just it's used 3 time so I thought I'd make a function to avoid replication

Reply:
Sure, could you rename the function to indicate it's an internal function (prefix with a `_`) and give it a proper function name? The current one is misleading as it only adjusts the lower bound

Reply:
sure